RELEASE.md

```diff
@@ -0,0 +1,81 @@
+# Briisk git Release
+
+## Table of Contents
+
+  1. [First release](#first-release)
+  1. [Bugfix to release](#bugfix-to-release)
+  1. [Hotfix to production release](#hotfix-to-production-release)
+
+## First Release
```

Comment:
Well, I am wondering because I suppose this flow is Bitbucket specific, and in near feature, we plan to move to GitHub. Maybe we should add both flow versions. WDYT?

Reply:
But it's a Git release flow, how it is different on Github?